solutions/python-igraph/visualize_cliques/visualize_cliques.py

```diff
@@ -0,0 +1,12 @@
+import igraph as ig
+import matplotlib.pyplot as plt
+
+g = ig.Graph.Famous('Zachary')
+g.es['width'] = 0.5
+cliques = g.cliques(4, 4)
+fig, ax = plt.subplots(3, 4)
+for i in range(len(cliques)):
```

Comment:
It might be a bit more Pythonic to do this like
```suggestion
for i, clique in enumerate(cliques):
```
and you can then immediately use `clique` instead of `cliques[i]`.